gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -452,13 +455,23 @@ public String getEndpoint() {
     return endpoint;
   }
 
+  @ObsoleteApi("Use getKeepAliveTimeDuration() instead")
+  public org.threeten.bp.Duration getKeepAliveTime() {
+    return toThreetenDuration(getKeepAliveTimeDuration());
+  }
+
   /** The time without read activity before sending a keepalive ping. */
-  public Duration getKeepAliveTime() {
+  public java.time.Duration getKeepAliveTimeDuration() {
     return keepAliveTime;
   }
 
+  @ObsoleteApi("Use getKeepAliveTimeoutDuration() instead")
```

Comment:
same comment as above - add this to the Javadoc as well

Reply:
Done